mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -668,6 +746,29 @@ impl UtxoRpcClientOps for NativeClient {
         )
     }
 
+    fn display_balances(&self, addresses: Vec<Address>, decimals: u8) -> UtxoRpcFut<Vec<(Address, BigDecimal)>> {
+        Box::new(
+            self.list_unspent_group(addresses.clone(), decimals)
+                .map(move |mut unspent_map| {
```

Comment:
This and some other mappings are nested too much. Please consider adding helpers so it will look like
```rust
fn unspent_map_to_balances(...)
....
self.list_unspent_group(addresses.clone(), decimals).map(unspent_map_to_balances)
```
Or maybe there's also another way to achieve simpler code.

Reply:
Done, thank you! Now it should look easier